Controller/HeadlessWebsiteController.php

```diff
@@ -69,11 +67,7 @@ protected function resolveStructure(StructureInterface $structure): array
      */
     protected function serializeData(array $data): string
     {
-        return $this->get('jms_serializer')->serialize(
-            $data,
-            'json',
-            (new SerializationContext())->setSerializeNull(true)
-        );
+        return json_encode($data, JSON_PRETTY_PRINT) ?: '';
```

Comment:
`JSON_PRETTY_PRINT` should only be done in `%kernel.debug%` or be removed completely.
 
Also I think `?: ''` should be changed to `?: {}`. Or we should use `JSON_THROW_ON_ERROR` which would require `PHP 7.3`.

Reply:
adjusted this to use the `JSON_THROW_ON_ERROR` flag 🙂